src/modules/Form/index.ts

```diff
@@ -0,0 +1,66 @@
+import Block from '../../utils/Block';
+import template from './form.pug';
+import * as formStyles from './form.module.scss';
+import { Button } from '../../components/Button';
+import { Input } from '../../components/Input';
+import { ValidationResult } from '../../utils/CustomValidation';
+
+interface FormProps {
+    action: string,
+    method: string,
```

Comment:
Я бы предложил сделать так
```suggestion
    action: string,
    method: 'GET' | 'POST' | 'PUT' | 'DELETE',
```
В таком случае TS нам подскажет, если мы (или кто-то другой, работающий с этим кодом) захотим что-то не то положить в это свойство

Reply:
Спасибо за рекомендацию